src/Apps/Order/Components/__tests__/SavedAddresses.jest.tsx

```diff
@@ -61,7 +61,7 @@ describe("Saved Addresses", () => {
           addressConnection: mockAddressConnection,
         }),
       })
-      const button = wrapper.find("Button[data-test='shippingButton']")
+      const button = wrapper.find("[data-test='shippingButton']").first()
```

Comment:
Curious about why `.first()` needs to be added here? I understand vaguely that .first() reduces all of the matched nodes and then wraps only the first node. Is this needed when checking a clickable link vs. a button? @rquartararo

Reply:
Yes, so there are multiple ways to go about this. The `data-test='shippingButton'` selector has two nodes because one of them is the `AddAddressButton` component, and the other is the button that the component renders. You can console.log the wrapper content using `wrapper.debug()` to see this more clearly. I could have also done `"SavedAddresses__AddAddressButton[data-test='shippingButton']"` to specifically select the component without using `first()`, as we did with the previous `Button` component. Not entirely sure which way is preferred as I've seen this done both ways in Force.

Reply:
One benefit of using `first()` in this case is that if we ever change the component name, we won't have to update the test again.